docs/kubernetes.adoc

```diff
@@ -86,7 +86,7 @@ For example **foo.bar.baz** will be converted to **FOO_BAR_BAZ**.
 | definitionsFile                     | String         | Any | Definitions file path
 | proxiedContainerPorts               | List           | Any | Comma Separated List following Pod:containerPort OR Pod:MappedPort:ContainerPort
 | routerHost                          | String         | Any | Define OpenShift router address thats is used to resolve the pod's address and to get its route
-| enableImageStreamDetection          | Bool (false)   | Any | Define if you want that Arquillian Cube apply Image Stream files (`*-is.yml`) before deploying
+| enableImageStreamDetection          | Bool (true)    | Any | Define if you want that Arquillian Cube apply Image Stream files (`*-is.yml`) before deploying
```

Comment:
I think it should say here that is false in case of kubernetes and true in case of OpenShift

Reply:
Looking at the code, it seems it's actually `true` always...? (I don't mind. I think it probably _should_be `true` by default, because it's required when running tests in a new Cube-managed namespace, which is the default behavior.)

Reply:
Yeah, I think it's an error to set it always to `true`, imagestream detection should not be enabled on Kubernetes. I was running tests with `mvn clean ...` so I didn't notice that if for some reason a `*-is.yml` file is on the build dir it's applied on Kubernetes too, even if Kubernetes does not support them.

Reply:
I might be wrong here. The `*-is.yml` files are only searched for in `CubeOpenShiftConfiguration`. Which, before this PR, was never supposed to be in place when using Kubernetes. With this PR, when you want to bypass all the OpenShift stuff when running against Kubernetes, I'm not sure.

Reply:
The problem is.. I looked at how to bypass completely the openshift stuff when I'm connected to kubernetes, but the Openshift `LoadableExtension` overrides some services before knowing if we are running on Kube or Openshift. Using the kubernetes client there to find the cluster type is not allowed, because the kubernetes client needs the configuration and the configuration depends on the cluster type (a typical chicken-egg problem).
There were already many calls to `kubernetesClient.isAdaptable(OpenshiftClient.class)` and they were meant to avoid doing openshift-related tasks on Kubernetes. I used the same reasoning on the resource locator.
So the solution towards I'm going is to make openshift stuff behave like Kubernetes stuff (many Openshift objects have a "is a" relationship with Kubernetes objects). So I'm trying to refactor a bit the configuration..